app/controllers/agents_controller.rb

```diff
@@ -273,6 +293,24 @@ def agent_params
         v
       end
     end
+    p[:identifiers]&.each_value do |identifier|
+      if identifier[:schemaAgency].downcase.eql?('orcid')
+        normalized_orcid = normalize_orcid(identifier[:notation])
+        if normalized_orcid
+          identifier[:notation] = normalized_orcid
+        else
+          identifier[:notation] = "invalid_orcid"
+        end
+      else
+        normalized_ror = normalize_ror(identifier[:notation])
+        if normalized_ror
+          identifier[:notation] = normalized_ror
+        else
+          identifier[:notation] = "invalid_ror"
```

Comment:
you can't use 'invalid_` to tell the API the this field is incorrect. 
The API and UI are two independent applications, that need to work without knowing something specific about each other.

Reply:
In this case, we have two options, 
1 - pass the invalid identifier to the back and do the validation in the back
2 - keep the validation in the UI side (too complexe since we already have the back error messages)

Reply:
3- don't validate